include/rdma/fabric.h

```diff
@@ -158,6 +158,8 @@ typedef struct fid *fid_t;
 #define FI_MATCH_COMPLETE	(1ULL << 31)
 
 #define FI_PEER_TRANSFER	(1ULL << 36)
+#define FI_MR_SINGLE_USE	(1ULL << 38)
```

Comment:
The global flag space is constrained.  Please move this into fi_domain.h as a memory registration flag.  The FI_MR_DMABUF flag below probably should have been located there as well.